package.json

```diff
@@ -110,7 +109,8 @@
     "webpack-merge": "^4.2.1"
   },
   "peerDependencies": {
-    "nightwatch": "^0.9.12"
+    "nightwatch": "^0.9.12",
+    "terra-aggregate-translations": "^1.0.0"
```

Comment:
I don't think this needs to be a peer dependency since the usage from this library doesn't require the cli to be available in the bin dir.

Reply:
updated!